src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -298,21 +298,21 @@ const UsersFragment = ({
 const styles = StyleSheet.create({
   fragmentContainer: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
   },
   noDataFoundContainer: {
     flex: 1,
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the background color for `noDataFoundContainer` has been changed from `COLORS.white` to `COLORS.almostBlack`. Make sure this change is intended and consistent with your design guidelines.
```diff
-    backgroundColor: COLORS.almostBlack
+    backgroundColor: COLORS.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->